### ISSUE TITLE
[L] Seed database with existing park name information

### ISSUE THREAD

**Issue by ashley-dhillon** [bcgov/parks-data-register#63]
We'll need to get access to park name info (RF & GE) and seed the database with this information in order to test out the API.

Data to seed the database can be found on Confluence - https://apps.nrs.gov.bc.ca/int/confluence/display/BCPRS/Park+Names
- check the Confluence page for specific columns to pull relevant data from.

Data will need to be organized by which columns to include, etc

Acceptance Criteria:

- Review spreadsheets to ensure alignment with data model
- Document mapping of spreadsheet columns to data model 
- Use mapping to bring over data from spreadsheet into data register database
- Testing: ensure database contains accurate data from spreadsheets

**Comment by danieltruong** [comment]
Dev, Test and Prod databases have been seeded. I have intentionally left out status because we do not know the mapping for it. Since we have an index on status, I am not able to leave it as an empty string. We will populate status when at a future time when the mapping for status is known.

**Comment by manuji** [comment]
@JLWade , quick clarification: Should these fields be there on the DynamoDB or is it a different list of fields? 



![Image](https://github.com/bcgov/parks-data-register/assets/65190263/2db5f30b-8357-46ec-aed1-c896cdcacc51)

**Comment by JLWade** [comment]
Yup those should all be there except Site (as that is further down in the backlog to add). However, they may not all be filled out with data - for example, many parks do not have a phonetic name or a audio clip.
 I think @danieltruong has added Status as current to everything as well. 
Does that help answer your question? @manuji

**Comment by ashley-dhillon** [comment]
@JLWade Testing

**Comment by manuji** [comment]
Thanks @JLWade .

**Comment by manuji** [comment]
Tested on PROD: **Pending**
* Missing the fields Audio Clip, and Legal Name Change Date, assuming the field "UpdateDate" on DynamoDB is not for Legal Name Change Date.
* Also display extra fields on DynamoDB which are not on the Excel sheet such as; "effectiveDate"

FYI @danieltruong

**Comment by JLWade** [comment]
moving to done as per demo day :)
